src/dotnet/commands/dotnet-new/CSharp_MSBuild/$projectName$.csproj

```diff
@@ -4,7 +4,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
-    <OutputPath>bin\$(Configuration)</OutputPath>
+    <OutputPath>bin\$(Configuration)\$(TargetFramework)</OutputPath>
```

Comment:
+1 to what @eerhardt and @livarcocc are saying. This doesn't seem like the right fix. I would expect you to get two levels of targetframework with this.